src/App.jsx

```diff
@@ -3,26 +3,82 @@ import './App.scss';
 import './styles/general.scss';
 import { TodoList } from './components/TodoList';
 import { CurrentUser } from './components/CurrentUser';
+import { getTodos } from './api/api';
 
 class App extends React.Component {
   state = {
     todos: [],
     selectedUserId: 0,
+    input: '',
+    select: 'all',
   };
 
+  componentDidMount() {
+    getTodos()
+    .then(todos => this.setState({todos: todos.data}));
+  }
+
+  selectUser = (newUserId) => (
+    this.setState({selectedUserId: newUserId})
```

Comment:
```suggestion
    this.setState({ selectedUserId: newUserId })
```
Интересно как линтер это пропустил